Wikipedia/Code/InsertMediaSettingsTableViewController.swift

```diff
@@ -96,6 +96,8 @@ final class InsertMediaSettingsTableViewController: UITableViewController {
                     }
                 }
 
+                static var unitName = WMFLocalizedString("insert-media-image-size-settings-px-unit-name", value: "px", comment: "Image size unit name, abbreviation for 'pixels'")
```

Comment:
Should be a format string with a plural in case any languages need to adjust pluralization or would have the word in front of the number

Reply:
"px" is a separate label and the number is in its own text field so that the user can see that whatever they input, it's always pixels - I think the only way to make it work as a format string would be having all of the text ("220 px") as text field's text? But then the user could think that they can/should input other units like "em" etc? I could take numbers only when I create the wikitext but then it could look like a bug when they see "px" in the wikitext but they input something else? 😬 not sure how to best handle this case
<img width="324" alt="Screen Shot 2019-04-24 at 5 35 26 PM" src="https://user-images.githubusercontent.com/9299317/56695550-5a354680-66b7-11e9-8aa1-775b1278e87d.png">

Reply:
Makes sense, there's not an easy way to have it as a text field, it'd have to be a separate screen for entry or maybe the unit moves to the Width & Height labels to be something like Width (pixels). Let's leave it as is for now (there are examples of `px` alone as a string on translatewiki) and discuss at task sync.